customer_satisfaction/streamlit_app.py

```diff
@@ -0,0 +1,90 @@
+from re import S
+import streamlit as st
+from inference.predict import predict 
+from PIL import Image
+import pandas as pd 
+
+import pickle 
+def main(): 
+    st.title("End to End Customer Satisfaction Pipeline with ZenML")
+
+
+    image = Image.open('_assets/high_level_overview.png')
```

Comment:
As discussed, this image should be cropped.

Reply:
yes, image is cropped